Index: app/src/main/java/com/example/bucketlist/model/ActivityModel.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/example/bucketlist/model/ActivityModel.java	(date 1603174826943)
+++ app/src/main/java/com/example/bucketlist/model/ActivityModel.java	(date 1603174826943)
@@ -0,0 +1,5 @@
+package com.example.bucketlist.model;
+
+public class ActivityModel {
+    String createdByUserID;
+}
